runtime/compiler/control/J9Options.cpp

```diff
@@ -2712,6 +2714,15 @@ J9::Options::fePostProcessJIT(void * base)
          }
       }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+#if defined(J9VM_OPT_JITSERVER)
+   if (!javaVM->internalVMFunctions->isJITServerEnabled(javaVM))
+#endif
+      {
+      _numAllocatedCompilationThreads = TR::CompilationInfo::MAX_CLIENT_USABLE_COMP_THREADS;
```

Comment:
This is server mode, right? In that case shouldn't this be `MAX_SERVER_USABLE_COMP_THREADS`

Reply:
No, this is for `!javaVM->internalVMFunctions->isJITServerEnabled(javaVM)` so it's only for the client; unless i'm misunderstanding what `isJITServerEnabled` is supposed to be checking for?

Reply:
Right, I missed the `!`